airflow/providers/amazon/provider.yaml

```diff
@@ -596,3 +597,4 @@ additional-extras:
   - name: pandas
     dependencies:
       - pandas>=0.17.1
+      - aiobotocore>=2.1.1
```

Comment:
In additional if user post-install this dependency then step should be

1. remove `boto3` and `botocore`
2. Install `aiobotocore` with appropriate version of `boto3` and `botocore`

Otherwise this error/warning happen

```console
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
boto3 1.26.79 requires botocore<1.30.0,>=1.29.79, but you have botocore 1.27.59 which is incompatible.
```

Reply:
do you mean in CI we should first remove botocore/boto3 and then install aiobototcore?  
yes, the above warning/error will come but since I'm running a single command to install just one package aiobotocore it will install it. let me know if you want I can add step in CI to remove botocore/boto3 and then install the  aiobotocore

Reply:
Not only in CI, we should mention about this in documentation otherwise we have tons of issues/discussion opened by users who want to use deferrable AWS operators and now have potentially broken system.